situp-api/src/main/java/com/amazon/situp/model/metrics/MetricNames.java

```diff
@@ -0,0 +1,26 @@
+package com.amazon.situp.model.metrics;
+
+public class MetricNames {
```

Comment:
Buffer metrics?

Reply:
Sure, will add those when I add the buffer metrics